cmd/buildah/common.go

```diff
@@ -158,3 +159,17 @@ func getDockerAuth(creds string) (*types.DockerAuthConfig, error) {
 		Password: password,
 	}, nil
 }
+
+func validateFlags(c *cli.Context, flags []cli.Flag) error {
+	for _, flag := range flags {
+		if reflect.TypeOf(flag).String() == "cli.StringFlag" {
```

Comment:
do we need to check booleans too?

Reply:
I don't think so. Booleans are not set that way.  --bool --foobar will not accidently set the bool to --foobar.

Reply:
No checks for string slice flags, though? We do have a lot of those.

Reply:
I added the StringSliceFlags check also.  I originally thought it would not happen but if you do 

buildah config --env --env foo=bar containerid 

It will give you a wacky error.  Fixed now.